morepath/view.py

```diff
@@ -44,7 +43,7 @@ def __init__(self, func, render, permission, internal):
         self.permission = permission
         self.internal = internal
 
-    def __call__(self, obj, request):
+    def __call__(self, app, obj, request):
```

Comment:
Ah, view is adjusted so it doesn't need the wrapper. I think this might be more difficult on my branch; I need to experiment.